apps/web/components/apps/layouts/AppsLayout.tsx

```diff
@@ -22,7 +22,7 @@ export default function AppsLayout({ children, actions, emptyStore, ...rest }: A
 
   if (session.status === "loading") return <></>;
   return (
-    <Shell {...rest} actions={actions?.("block")} hideHeadingOnMobile>
+    <ShellMain {...rest} actions={actions?.("block")} hideHeadingOnMobile>
```

Comment:
Have been using this strategy for Routing Forms navigations. Using `ShellMain` here allows configuring shell on per route basis. `MainLayout` renders `Shell` without main content